src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -120,39 +111,39 @@ const AudioInputVFControl: React.FC<Props> = ({
     const dropdownOptions: ReactNode[] = audioInputDevices.map((device) => (
       <PopOverItem
         key={device.deviceId}
-        children={<span>{device.label}</span>}
         checked={isOptionActive(selectedDevice, device.deviceId)}
         onClick={async (): Promise<void> => {
           if (isVoiceFocusChecked && !isLoading) {
             setIsLoading(true);
-            const receivedDevice = audioInputSelectionToDevice(device.deviceId);
+            const receivedDevice = device.deviceId;
             const currentDevice = await addVoiceFocus(receivedDevice);
             await meetingManager.selectAudioInputDevice(currentDevice);
             setIsLoading(false);
           } else {
             await meetingManager.selectAudioInputDevice(device.deviceId);
           }
         }}
-      />
+      >
+        <span>{device.label}</span>
+      </PopOverItem>
     ));
 
     if (isVoiceFocusSupported) {
       const vfOption: ReactNode = (
         <PopOverItem
           key="voicefocus"
-          children={
-            <>
-              {isLoading && <Spinner width="1.5rem" height="1.5rem" />}
-              {isVoiceFocusEnabled ? voiceFocusOnLabel : voiceFocusOffLabel}
-            </>
-          }
           checked={isVoiceFocusEnabled}
           disabled={isLoading}
           onClick={() => {
             setIsLoading(true);
             setIsVoiceFocusChecked((current) => !current);
           }}
-        />
+        >
+          <>
```

Comment:
May be we can remove this Fragment wrapping?

Reply:
Removing this will lead to type error. Have to wrap them as a single child.
```jsx
<>
  {isLoading && <Spinner width="1.5rem" height="1.5rem" />}
  {isVoiceFocusEnabled ? voiceFocusOnLabel : voiceFocusOffLabel}
</>
```